libs/designsystem/src/lib/components/action-sheet/action-sheet.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { fakeAsync, tick } from '@angular/core/testing';
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import { MockComponent } from 'ng-mocks';
+
+import { PlatformService } from '../..';
```

Comment:
It's a bit hard to see where this comes from - consider more specific path...

Reply:
fixed